Userland/Libraries/LibRegex/RegexParser.cpp

```diff
@@ -1784,10 +1792,17 @@ bool ECMA262Parser::parse_nonempty_class_ranges(Vector<CompareTypeAndValuePair>&
                 if (try_skip("-"))
                     return { CharClassRangeElement { .code_point = '-', .is_character_class = false } };
 
-                Unicode::Property property {};
+                PropertyEscape property {};
                 bool negated = false;
-                if (parse_unicode_property_escape(property, negated))
-                    return { CharClassRangeElement { .property = property, .is_negated = negated, .is_character_class = true, .is_property_escape = true } };
+                if (parse_unicode_property_escape(property, negated)) {
+                    return property.visit(
+                        [&](Unicode::Property property) -> CharClassRangeElement {
+                            return { CharClassRangeElement { .property = property, .is_negated = negated, .is_character_class = true, .is_property = true } };
```

Comment:
The first set of braces are redundant, and you can omit one of the `CharClassRangeElement`'s.
(previous lines need the extra braces because they're initialising an `Optional<CharClassRangeElement>`, which is being implicitly constructed here)

Reply:
Copy-paste, always a good idea